controllers/articles.controller.js

```diff
@@ -26,4 +27,25 @@ const getCommentsByArticleId = (req,res,next) => {
     .catch(next)
 }
 
-module.exports = {getArticleById, getArticles, getCommentsByArticleId}
\ No newline at end of file
+const postCommentsByArticleId = (req,res,next) => {
+
+    const articleId = req.params.article_id
+    const newComment = req.body
+
+    if (!newComment.body || !newComment.username || Object.keys(newComment).length !==2) {
+        return next({status:400, msg: 'Invalid Request Body'})
+    }
```

Comment:
Probably better to have this part in your model and actually throw an error back (using Promise.reject) instead of invoke next with an object